Initialize-PowershellForDevelopment.ps1

```diff
@@ -50,12 +50,19 @@ function Install-RequiredPackageProvider {
 }
 
 function Install-RequiredModules {
-    $installedModules = Get-InstalledModule | ? -Property Name -eq "Invoke-MsBuild"
+    $msbuild = Get-InstalledModule | ? -Property Name -eq "Invoke-MsBuild"
 
-    if ($installedModules -eq $null) {
-        Write-Host "Installing required modules"
+    if ($null -eq $msbuild) {
+        Write-Host "Installing Invoke-MsBuild"
         Install-Module -Name Invoke-MsBuild -Scope CurrentUser -MinimumVersion 2.6.0 -Force | Out-Null
     }
+
+    $pester = Get-InstalledModule | ? -Property Name -eq "Pester"
```

Comment:
Do we have to force Pester to be installed for all initdev calls?  Could this be wrapped in an entry point function to execute powershell tests, similar to how we have the Newman dependency wrapped in Invoke-PostmanTests?  Don't get me wrong, I like the idea of finally having powershell tests, but not sure it should be a default dependency for someone just cloning the code and running initdev.

Reply:
moved pester install to initdev task